fix: unused event PaneActionRouter.ToggleFullscreenRequested

CS0067 warning on every build:

```
PaneActionRouter.cs(69,32): warning CS0067: The event 'PaneActionRouter.ToggleFullscreenRequested' is never used
```

Either wire up the event or remove it.

--- a/windows/Ghostty/Input/PaneActionRouter.cs
+++ b/windows/Ghostty/Input/PaneActionRouter.cs
@@ -83,6 +83,9 @@ internal sealed class PaneActionRouter
             case PaneAction.ToggleCommandPalette:
                 CommandPaletteToggleRequested?.Invoke(this, EventArgs.Empty);
                 return;
+            case PaneAction.ToggleFullscreen:
+                ToggleFullscreenRequested?.Invoke(this, EventArgs.Empty);
+                return;
         }
 
         var pane = _tabs.ActiveTab.PaneHost;

